mailbox/scanning-search/src/test/java/org/apache/james/mailbox/store/search/SimpleMessageSearchIndexTest.java

```diff
@@ -54,192 +54,192 @@ protected void initializeMailboxManager() {
     public void sortOnDisplayToShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void flagIsUnSetShouldReturnUidOfMessageNotMarkedAsRecentWhenUsedWithFlagRecent() throws MailboxException {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void bodyContainsShouldReturnUidOfMessageContainingTheApproximativeText() throws MailboxException {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void headerDateBeforeShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void sortOnSentDateShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void addressShouldReturnUidHavingRightRecipientWhenToIsSpecified() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void sortOnToShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void sortOnDisplayFromShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void revertSortingShouldReturnElementsInAReversedOrder() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void headerDateAfterShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void mailsContainsShouldIncludeMailHavingAttachmentsMatchingTheRequest() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void sortOnSubjectShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void modSeqGreaterThanShouldReturnUidsOfMessageHavingAGreaterModSeq() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void notShouldReturnResultsThatDoNotMatchAQuery() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void headerDateOnShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void sortOnSizeShouldWork() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void sortShouldOrderMessages() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void multimailboxSearchShouldReturnUidOfMessageWithExpectedFromInTwoMailboxes() throws MailboxException {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void searchWithTextShouldReturnMailsWhenTextBodyMatchesAndNonContinuousWords() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void searchWithTextShouldReturnMailsWhenHtmlBodyMatchesAndNonContinuousWords() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void searchWithTextShouldReturnMailsWhenTextBodyWithExtraUnindexedWords() throws Exception {
     }
 
-    @Ignore
+    @Disabled
     @Override
     public void searchWithTextShouldReturnMailsWhenHtmlBodyMatchesWithStemming() throws Exception {
     }
 
-    @Ignore
+    @Disabled
```

Comment:
While we are at it, is there a ticket for these failed tests? 

Can we put it in the description?

Reply:
Good point, will try to look for it...